extra-utils/onefetch/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=onefetch
+PKGSEC="utils"
+PKGDEP="git"
+BUILDDEP="rustc llvm"
+PKGDES="A command-line Git information tool"
```

Comment:
Hard to imagine what "Git information" means.

Reply:
Consider "Git information" -> "Git repository information".